replica/database.hh

```diff
@@ -1316,6 +1316,10 @@ public:
         return _replication_strategy;
     }
 
+    bool uses_tablets() const {
+        return _replication_strategy->uses_tablets();
+    }
+
```

Comment:
We'll have to rethink it when we go vnode->tablet migration, but that applies to many things.